### PR TITLE
Move fast and toggle help box

### PR DESCRIPTION
**Pull request by uhunkler**
Hi Vincent,

I was missing the ability to move fast, so I added fast moving on shift-down similar to Hubs. And because the help box on the page is getting bigger I added a toggle button to be able close/open the help information.

These extensions I added only in the playground.

What do you think?

### CODE REVIEW

**Review by vincentfretin — COMMENTED** [review]
Thanks, but I would prefer we add this feature by default in aframe-extras keyboard-controls, using a x2 multiplier, probably putting 2 here https://github.com/c-frame/aframe-extras/blob/a1be8e84af4ada91addf5c7eee5821752d1cb8d4/src/controls/keyboard-controls.js#L60
See https://github.com/c-frame/aframe-extras/issues/426
I'll be happy if you can make a PR there.

File: examples/playground/index.html [hunk 1]
```diff
@@ -116,25 +134,53 @@
...
     </div>
+    </div>
+
+    <script>
+      const helpbox = document.querySelector("#help-box");
+      const btn = helpbox.querySelector("button.close");
+      const content = helpbox.querySelector('#help-content');
+      btn.addEventListener("click", () => {
+        if (content.hasAttribute("hidden")) {
+          content.style="display:block;";
```
Comment (vincentfretin) [inline examples/playground/index.html:173]: Space around '=' missing.
I should put in the repo husky/lint-staged with prettier to format the file on commit. If you use vscode, please install prettier addon to format the file.

**Comment by uhunkler** [comment]
I added a PR for »[keyboard-controls] Press shift to run #426« in aframe-extras as you proposed.

**Review by vincentfretin — APPROVED** [review]
Thanks!

**Comment by vincentfretin** [comment]
I updated to aframe-extras 7.4.0 to have shift to run in https://github.com/c-frame/aframe-gltf-model-plus/commit/465299d57b0b082ac156b691dc92a4ebdac3594b

**Pull request merged**